infrastructure/assumptions/cookie.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<title>cookies work in default browse settings</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://html.spec.whatwg.org/#dom-document-cookie">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<script>
+  test(() => {
+    document.cookie = "name=test_cookie";
+    assert_not_equals(document.cookie.match(/name=test_cookie/), null);
+  });
```

Comment:
Can you add a cleanup function here to remove all cookies?

Arguably we actually want something that ensures that cookies work across navigation, but I'll let others decide whether that's worth having here?

Reply:
@gsnedders, I've addressed the suggestion in new commit, PTAL.